Source/Shaders/ModelExperimental/MaterialStageFS.glsl

```diff
@@ -23,7 +23,7 @@ vec3 computeNormal(ProcessedAttributes attributes)
     vec3 ng = attributes.normalEC;
 
     vec3 normal = ng;
-    #ifdef HAS_NORMAL_TEXTURE
+    #if defined HAS_NORMAL_TEXTURE && !defined USE_WIREFRAME
```

Comment:
For consistency with existing code, let's put parentheses, e.g. `defined(HAS_NORMAL_TEXTURE)`